docs/developer-guide/ksqldb-reference/scalar-functions.md

```diff
@@ -960,12 +960,12 @@ for example, when the string is `NULL` or it does not contain valid JSON,
 or the JSON value is not an object.
 
 ```sql title="Examples"
-JSON_RECORDS("{\"a\": \"abc\", \"b\": { \"c\": \"a\" }, \"d\": 1}") => {"a": "\"abc\"", "b": "{ \"c\": \"a\" }", "d": "1"}
-JSON_RECORDS("{}") => []
-JSON_RECORDS("[]") => NULL
-JSON_RECORDS("123") => NULL
+JSON_RECORDS('{\"a\": \"abc\", \"b\": { \"c\": \"a\" }, \"d\": 1}') => {"a": '\"abc\"', "b": '{ \"c\": \"a\" }', "d": "1"}
+JSON_RECORDS('{}') => []
```

Comment:
The output I see from the CLI in this case is:
```
{}
```

Reply:
This is a typo, `{}` is indeed a correct return value here.